optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/statistic/PureSubSingleStatistic.java

```diff
@@ -39,7 +39,8 @@
         PickedMoveTypeBestScoreDiffSubSingleStatistic.class,
         PickedMoveTypeStepScoreDiffSubSingleStatistic.class
 })
-public abstract class PureSubSingleStatistic<P extends StatisticPoint> extends SubSingleStatistic<P> {
+public abstract class PureSubSingleStatistic<Solution_, P extends StatisticPoint>
```

Comment:
Will do. Although I'm sure you see that it's been like this before, I just didn't change it. :-)